src/frontends/pytorch/src/op/conv_transposend.cpp

```diff
@@ -0,0 +1,62 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset10.hpp"
```

Comment:
Could you instead of including opset include only headers of needed operations?

Reply:
updated